.github/workflows/publish.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
oh, wait, this is updating the `checkout` action, not the `cache` action below 😅

Reply:
Fixed.